examples/step-47/doc/results.dox

```diff
@@ -13,7 +13,7 @@ $Q_3$; and 5, 4, 3 for $Q_4$, respectively.
 
 From the literature, it is not immediately clear what
 the penalty parameter $\gamma$ should be. For example,
-@cite Brenner2009 state that it needs to be larger than one, and
+@cite Brenner2010 state that it needs to be larger than one, and
```

Comment:
Do you mean this? This fix in the ``examples/step-47/doc/results.dox`` was already included in this PR.